frontend/app/enrollment/components/selected-service-request-view.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card'
+import useEnrollmentStore from '@/lib/store/enrollment-store'
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from '@/components/ui/tooltip'
+import { InfoCircledIcon } from '@radix-ui/react-icons'
+
+export default function SelectedServiceRequestView() {
+
+    const { serviceRequest } = useEnrollmentStore()
+
+    return (
+        <Card className='mt-5 bg-primary text-gray-100'>
+            <CardHeader>
+                <CardTitle>
+                    <div className='flex items-center'>
+                        <TooltipProvider>
+                            <Tooltip>
+                                <TooltipTrigger asChild>
+                                    <InfoCircledIcon className='h-6 w-6 cursor-help' />
+                                </TooltipTrigger>
+                                <TooltipContent>
+                                    <p>A Task refers to the outgoing order</p>
+                                </TooltipContent>
+                            </Tooltip>
+                        </TooltipProvider>
+                        <p>&nbsp;Task information</p>
+                    </div>
+                </CardTitle>
+            </CardHeader>
+            <CardContent>
+                <div className='grid grid-cols-4 gap-4'>
+                    <div><b>Task: </b></div>
```

Comment:
I don't do much front-end, but should we do styling with CSS instead of `<b>`'s?